app/fixtures/fixtures.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+  listOfMediaFilesForPOI: [
+    {
+      "mediaId": "d07d6ab3-4e3f-44ce-accc-b3efc96b3f04",
+      "ipfs": "QmVXmy59f5QqKDGyhpHbRiGnba5SfHaooDPCWhd8Xtgwz2",
```

Comment:
@almereyda In case of IPFS stored media files. I would suggest that we add the **url** to the JSON payload of such an element. That way, the code of the editor would just show the image by url and not need to do any specific logic depending how and where the image is stored...

As I understand, an image stored in the IPFS network is also accessible via a cotidian HTTP url, right?

Reply:
Yes, we should include a mandatory URL field for any media item.